live-build/misc/live-build-hooks/92-ova-machine-image.binary

```diff
@@ -74,7 +74,7 @@ find "$OVA_DIRECTORY" -type f | while read -r file; do
 	NAME=$(basename "$file")
 	HASH=$(sha1sum "$file" | awk '{print $1}')
 	cat <<-EOF >>"$OVA_DIRECTORY/$APPLIANCE_VARIANT.mf"
-	SHA1($NAME)= $HASH
+		SHA1($NAME)= $HASH
```

Comment:
Is this change intentional?

Reply:
Do you know why this wasn't failing the automated checks? We don't run shfmt against this file via the automated PR checks?

Reply:
It's an issue with shfmt version: v2.30 on our docker image vs v.2.40 on OSX through brew.
I'll update the docker image to use 2.40

Reply:
Ah, OK. Thanks, that makes sense.